### PR TITLE
Initial support for Skupper operator

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: pkg/framework/pod.go [hunk 1]
```diff
@@ -180,6 +182,25 @@ func (c *ContextData) GetLogs(podName string) (string, error) {
 	return buf.String(), nil
 }
 
+func (c *ContextData) WaitForPodStatus(podName string, status v1.PodPhase, timeout time.Duration, interval time.Duration) (*v1.Pod, error) {
+	var pod *v1.Pod
+	var err error
+	ctx, cancel := context.WithTimeout(context.TODO(), timeout)
+	defer cancel()
+	for pod == nil {
```
Comment (nicob87) [inline pkg/framework/pod.go:190]: why a for? on a RetryWithContext, that already is doing the retry?

Comment (fgiorgetti) [reply pkg/framework/pod.go:190]: Not sure why I left it there, let me try running it again and see
if this is needed, but seems like useless to me.

Comment (fgiorgetti) [reply pkg/framework/pod.go]: @nicob87 it was probably a leftover from tests I was doing. I have removed the for statement from both pod.go and service.go. Thanks for the review.

**Comment by nicob87** [comment]
Approved! if travis-ci tests pass

**Comment by fgiorgetti** [comment]
@nicob87 this is right, I missed that... I will update the travis script to install skupper before running e2e tests.

**Pull request merged**